### ISSUE TITLE
[feature] ignore aria-hidden labels as an option

### ISSUE THREAD

**Issue by jstjoe** [DhruvParikh1/markdownload-extension-updated#44]
Thank you for making this extension! I've been recommending it to lots of people and it fills an important gap. And I have a suggestion for how you can make it 5x more useful for me, and probably others too.

Currently it seems to skip content which is in the HTML but marked with aria-hidden=true. This means that many websites which use collapsible elements (click-to-expand FAQs, for example) cannot be converted to markdown by this extension because many of these websites only allow one item to be expanded at a time.

It would be AMAZING if I could uncheck a "skip hidden content" box and make the extension include all of the hidden text / elements as well. 🙏

Thanks again!

**Comment by DhruvParikh1** [comment]
Hi @jstjoe,

Thanks for the kind words and for recommending the extension! I'm glad to hear it's been useful for you.

I completely agree with the suggestion, I've run into the same problem with collapsible elements myself so I will look into implementing that option soon. Keep an eye out.

Cheers
